jenkins/DeployAuthn.jenkinsfile

```diff
@@ -0,0 +1,243 @@
+
+				// set ECR account number
+				env.ECR_ACCOUNT = sh(label: 'Get ECR account', returnStdout: true, script: "aws secretsmanager get-secret-value --region $REGION --secret-id bento/$PROJECT/$ENV --query SecretString --output text | jq -r '.ecr_account'").trim()
+
+				// set repo URL
+				env.REPO_URL = "${ECR_ACCOUNT}.dkr.ecr.${REGION}.amazonaws.com/${ECR_REPO}"
+
+				// set template file name
+				env.TEMPLATE_FILE="ctdc-deployments/aws/${APP}_task.json"
+
+				env.NEO4J_HOST= sh(label: 'Get Neo4j host', returnStdout: true, script: "aws secretsmanager get-secret-value --region $REGION --secret-id bento/$PROJECT/$ENV --query SecretString --output text | jq -r '.neo4j_ip'").trim()
+				
+				env.MYSQL_DB_NAME="ctdc"
+				env.MYSQL_DB_PORT="3306"
+
+			}
+
+ 		}
+
+  	}
+
+  	stage('Update Task Definition'){
+
+ 		steps {
+
+			script {
+
+			    sh label: 'Task-Definition-Update', script: '''#!/bin/bash
+
+				# create new revision for $PROJECT-$APP Task Definition
+				echo "Updating Task Defintion to Use: $ECR_REPO:$IMAGE_TAG"
+				ACCOUNT_ID=$(aws sts get-caller-identity --query "Account" --output text)
+				IMAGE_ID="$REPO_URL:$IMAGE_TAG"
+				DATE=$(date '+%m/%d/%Y')
+
+				sed -i "s|{{account_id}}|$ACCOUNT_ID|" $TEMPLATE_FILE
+				sed -i "s|{{image_id}}|$IMAGE_ID|" $TEMPLATE_FILE
+				sed -i "s|{{image_tag}}|$IMAGE_TAG|" $TEMPLATE_FILE
+				sed -i "s|{{environment}}|$ENV|" $TEMPLATE_FILE
+				sed -i "s|{{program}}|$PROGRAM|" $TEMPLATE_FILE
+				sed -i "s|{{project}}|$PROJECT|" $TEMPLATE_FILE
+				sed -i "s|{{domain_name}}|$DOMAIN_NAME|" $TEMPLATE_FILE
+				sed -i "s|{{neo4j_host}}|$NEO4J_HOST|" $TEMPLATE_FILE
+				sed -i "s|{{mysql_database_name}}|$MYSQL_DB_NAME|" $TEMPLATE_FILE
+				sed -i "s|{{mysql_database_port}}|$MYSQL_DB_PORT|" $TEMPLATE_FILE
+				sed -i "s|{{sumo_collector_endpoint}}|$SUMO_COLLECTOR_ENDPOINT|" $TEMPLATE_FILE
+				sed -i "s|{{sumo_collector_token}}|$SUMO_COLLECTOR_TOKEN|" $TEMPLATE_FILE
+				sed -i "s|{{date}}|$DATE|" $TEMPLATE_FILE
+
+				aws ecs register-task-definition --cli-input-json file://$TEMPLATE_FILE
+
+				'''
+
+			}
+
+ 		}
+
+  	}
+
+    stage('Update Service'){
+
+ 		steps {
+
+ 			script {
+
+			    sh label: 'Service-Update', script: '''#!/bin/bash
+
+				# Redeploy Service Using The Latest Task Definition
+				echo "Updating Service: $SERVICE"
+
+				TASK_REVISION=$(aws ecs describe-task-definition --task-definition $SERVICE | egrep "revision" | awk '{print $2}' | sed 's/,//')
+				aws ecs update-service --cluster $CLUSTER --region $REGION --service $SERVICE --task-definition $SERVICE:$TASK_REVISION --desired-count 1
+
+				'''
+
+			}
+
+ 		}
+
+  	}
+
+	stage('verify deployment'){
+
+		steps {
+
+ 			script {
+
+			    sh label: 'Verify-Deploy', script: '''#!/bin/bash
+
+                # wait untiil the service is stable
+				aws ecs wait services-stable --cluster $CLUSTER --region $REGION --service $SERVICE
+
+				'''
+
+			}
+
+ 		}
+
+  	}
+	
+	stage('Add Production Tag'){
```

Comment:
please update the deploy files to use the latest prod tagging block from the templates in datacommons-devops